inscriptions/ISic001195.xml

```diff
@@ -190,22 +195,21 @@
     </facsimile>
     <text>
         <body>
-            <div type="edition" xml:space="preserve" subtype="PHI" n="PHI140679">
+            <div type="edition" xml:space="preserve" xml:lang="grc" resp="#MH">
                 <ab>
-                    <lb n="1"/>
-
-
-                    Ἀρτεμιδ
-                    <supplied reason="omitted">ώ</supplied>
-                    <unclear>ρ</unclear>ο
-                    <supplied reason="lost" cert="low">υ</supplied>
-                    <lb n="2"/>Ψυχ
-                    <supplied reason="lost">— — — — — —</supplied>
-                    <lb n="3"/>
+                    <lb n="1"/><orig>Ἀρτεμιδ</orig> <gap reason="lost" extent="unknown" unit="character"/>
```

Comment:
this can be `<name><seg part="I">Ἀρτεμιδ</seg><gap reason="lost" extent="unknown" unit="character"/></name>`